tests/unit/cocoon/test_utilities.py

```diff
@@ -777,16 +803,16 @@ def test_return_nested_model(
 
         :return: None
         """
-        nested_model = cocoon.utilities.return_nested_model(
+        nested_model = cocoon.utilities.check_nested_model(
             required_attribute_properties
         )
 
-        self.assertEqual(first=nested_model, second=expected_outcome)
+        self.assertTrue(expr=nested_model is expected_outcome)
```

Comment:
its better to use `assertEquals` as it gives you a better error message when it fails

I would probably have 2 tests, 1 for passes and one for fails

Reply:
Updated to use assertEquals. As it is not raising an error I am going to keep them in a single test.